.github/workflows/main.yml

```diff
@@ -1017,266 +766,161 @@ jobs:
         with:
           name: ${{ env.FULL_VERSION_NAME }}
           path: /tmp/artifacts/aar/${{ env.FULL_VERSION_NAME }}.aar
 
-  publish-libvcx:
-    runs-on: ubuntu-20.04
-    needs: [ workflow-setup, build-image-indypool, build-image-libvcx ]
-    env:
-      CACHE_KEY_LIBVCX: ${{needs.workflow-setup.outputs.CACHE_KEY_LIBVCX}}
-      DOCKER_IMG_NAME_LIBVCX: ${{needs.workflow-setup.outputs.DOCKER_IMG_NAME_LIBVCX}}
-      PUBLISH_VERSION: ${{needs.workflow-setup.outputs.PUBLISH_VERSION}}
-    steps:
-      - name: Git checkout
-        uses: actions/checkout@v2
-
-      - name: Load libvcx image cache
-        id: load-cached-libvcx-image
-        uses: actions/cache@v2
-        with:
-          path: /tmp/imgcache
-          key: ${{ env.CACHE_KEY_LIBVCX }}
-      - name: If no cached image found
-        if: steps.load-cached-libvcx-image.outputs.cache-hit != 'true'
-        run: echo "ERROR == Expected to find image from cache $CACHE_KEY_LIBVCX"; exit -1
-      - name: Load image from cache
-        run: docker load < /tmp/imgcache/img_libvcx.rar
-
-      - name: Verify libvcx image were loaded
-        run: |
-          docker image ls --format "{{.Repository}}:{{.Tag}}" | grep "$DOCKER_IMG_NAME_LIBVCX" || { echo "Image $DOCKER_IMG_NAME_LIBVCX was not found!" ; exit 1; }
-
-      - name: Docker Login
-        uses: azure/docker-login@v1
-        with:
-          login-server: ghcr.io
-          username: $GITHUB_ACTOR
-          password: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Publish image
-        run: |
-          if [[ "$PUBLISH_VERSION" ]]
-          then
-            IFS=$':' read -a arr <<< $DOCKER_IMG_NAME_LIBVCX
-            DOCKER_IMG_NAME_TAGLESS=${arr[0]}
-            GITHUB_REPOSITORY_LOWERCASE=`echo $GITHUB_REPOSITORY | awk '{print tolower($0)}'`
-            REMOTE_DOCKER_IMG_NAME_LIBVCX="ghcr.io/${GITHUB_REPOSITORY_LOWERCASE}/${DOCKER_IMG_NAME_TAGLESS}:${PUBLISH_VERSION}"
-            echo "Releasing libvcx docker image version $PUBLISH_VERSION, tagged $REMOTE_DOCKER_IMG_NAME_LIBVCX"
-            docker tag "$DOCKER_IMG_NAME_LIBVCX" "$REMOTE_DOCKER_IMG_NAME_LIBVCX"
-            docker push "$REMOTE_DOCKER_IMG_NAME_LIBVCX"
-          else
-             echo "New version was not defined, skipping release."
-          fi
-
+  #  ##########################################################################################
+  #  ############################   NPMJS PUBLISHING   #######################################
+  
   publish-node-wrapper:
     runs-on: ubuntu-20.04
-    needs: [ workflow-setup, build-image-indypool, build-image-libvcx-tester, test-libvcx-image-general_test, test-libvcx-image-pool-tests, test-libvcx-image-agency-pool-tests, test-node-wrapper, test-integration-node-wrapper ]
+    needs: [ workflow-setup, build-docker-libvcx-tester, test-libvcx-image-general_test, test-libvcx-image-pool-tests, test-libvcx-image-agency-pool-tests, test-node-wrapper, test-integration-node-wrapper ]
```

Comment:
Small nitpick, but perhaps we should settle on using either the single line or multiline list notation

Reply:
Personally, I find this particular list quite long :)

Reply:
touche